action.yml

```diff
@@ -16,6 +16,12 @@ inputs:
     description: '(Optional) Transforms environment variable name. Options: uppercase, lowercase, none. Default value: uppercase.'
     required: false
     default: 'uppercase'
+  auto-select-family-attempt-timeout:
+    description: '(Optional) Timeout in milliseconds for each attempt to retrieve a secret from a secret family. Default value: 1000'
```

Comment:
This is [node documentation talking about `autoSelectFamily`](https://nodejs.org/api/net.html). Would you mind trying to use it as base for the description of this field (and documentation). Try to summarize what it does and when should customer use it (also you have typo in select family vs secret family).